honeycombio/resource_burn_alert_test.go

```diff
@@ -135,10 +135,7 @@ resource "honeycombio_burn_alert" "test" {
 
   recipient {
     id = honeycombio_pagerduty_recipient.test.id
-
-    notification_details {
-      pagerduty_severity = "critical"
-    }
+    // default value is critical
```

Comment:
now that you removed the details block, "value" is unclear. Maybe just expand this comment a bit?